app/views/pages/_date_chart.html.slim

```diff
@@ -0,0 +1,49 @@
+canvas#categories-pie data-titles="#{@data[:categories_titles]}" data-amounts="#{@data[:categories_amounts]}" data-colors="#{@data[:charts_colors]}"
+
+javascript:
+  var categoriesPie = document.getElementById('categories-pie').getContext('2d');
+  var categories_titles = JSON.parse($('#categories-pie').attr('data-titles'));
+  var categories_amounts = JSON.parse($('#categories-pie').attr('data-amounts'));
+  var charts_colors = JSON.parse($('#categories-pie').attr('data-colors'));
+  
+  Chart.defaults.global.defaultFontSize = 18;
+  Chart.defaults.global.defaultFontColor = '#777';
+
+  var massPopChart = new Chart(categoriesPie, {
+    type:'pie',
+    data:{
+      labels: categories_titles,
+      datasets:[{
+        label:'Amount',
+        data: categories_amounts,
+        backgroundColor: charts_colors,
+        borderWidth:1,
+        borderColor:'#777',
+        hoverBorderWidth:3,
+        hoverBorderColor:'#000'
+      }]
+    },
+    options:{
+      title:{
+        display: false
+      },
+      legend:{
+        display:true,
+        position:'right',
+        labels:{
+          fontColor:'#000'
+        }
+      },
+      layout:{
+        padding:{
+          left:50,
+          right:0,
+          bottom:0,
+          top:0
+        }
+      },
+      tooltips:{
+        enabled:true
+      }
+    }
+  });
```

Comment:
@marderer why didn't you use `src/javascript/charts.js`? If you need some data from the server than just use `data-` html attribute and read it via jQuery

Reply:
@AleksSenkou We should use `date-` attributes. It from trello task.